### PR TITLE
Working on RHEL-07-031000.

### PR DESCRIPTION
**Pull request by mikerenfro**

### CODE REVIEW

File: tasks/fix-cat2.yml [hunk 1]
```diff
@@ -1292,16 +1292,31 @@
...
+    path: /etc/rsyslog.conf
+    state: present
+    block: |
+      $ActionQueueFileName rhel-07-031000 # unique name prefix for spool files
+      $ActionQueueMaxDiskSpace 1g   # 1gb space limit (use as much as possible)
+      $ActionQueueSaveOnShutdown on # save messages to disk on shutdown
+      $ActionQueueType LinkedList   # run asynchronously
+      $ActionResumeRetryCount -1    # infinite retries if host is down
+      # remote host is: name/ip:port, e.g. 192.168.0.1:514, port optional
+      *.* @@{{ rhel7stig_log_aggregation_server }}
```
Comment (jamescassell) [inline tasks/fix-cat2.yml]: I  think if you add ' | default("") to this variable, it will fix the error.

Comment (mikerenfro) [reply tasks/fix-cat2.yml]: The weird thing is the idempotence test is what's failing, [due to ssh key changes](https://travis-ci.org/MindPointGroup/RHEL7-STIG/builds/357455111#L4066) that should have nothing to do with this commit.

Comment (jamescassell) [reply tasks/fix-cat2.yml]: ok, i didn't actually look at the output...

**Comment by shepdelacreme** [comment]
Reran Travis and it came back clean. Probably just a latent CI error. Needs rebase.

**Pull request merged**